src/rdkafka_broker.c

```diff
@@ -5208,24 +5209,54 @@ static int rd_kafka_broker_name_parse(rd_kafka_t *rk,
         return 0;
 }
 
+/**
+ * @brief Add a broker from a string of type "[proto://]host[:port]" to the list
+ * of brokers. *cnt is increased by one if a broker was added, else not.
+ */
+static void rd_kafka_find_or_add_broker(rd_kafka_t *rk,
+                                        rd_kafka_secproto_t proto,
+                                        const char *host,
+                                        uint16_t port,
+                                        int *cnt) {
+        rd_kafka_broker_t *rkb = NULL;
+
+        if ((rkb = rd_kafka_broker_find(rk, proto, host, port)) &&
+            rkb->rkb_source == RD_KAFKA_CONFIGURED) {
```

Comment:
Remove braces. 

You can run `make style-check-changed` to figure out styling issues with the changed code. and `make style-fix-changed` to fix those issues.

Reply:
We need the braces, otherwise during we get the warnings

```
rdkafka_broker.c:5223:17: warning: incompatible integer to pointer conversion assigning to 'rd_kafka_broker_t *' (aka 'struct rd_kafka_broker_s *') from 'int' [-Wint-conversion]
        if (rkb = rd_kafka_broker_find(rk, proto, host, port) &&
                ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rdkafka_broker.c:5223:17: warning: using the result of an assignment as a condition without parentheses [-Wparentheses]
        if (rkb = rd_kafka_broker_find(rk, proto, host, port) &&
            ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rdkafka_broker.c:5223:17: note: place parentheses around the assignment to silence this warning
        if (rkb = rd_kafka_broker_find(rk, proto, host, port) &&
                ^
            (
rdkafka_broker.c:5223:17: note: use '==' to turn this assignment into an equality comparison
        if (rkb = rd_kafka_broker_find(rk, proto, host, port) &&
                ^
                ==
```